package.json

```diff
@@ -12,53 +12,63 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel 'lint:!(fix)'",
     "lint:alex": "alex {content,post-templates}/**/*.md",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "lint:md": "markdownlint {content,post-templates}/**/*.md",
     "start": "ember serve",
-    "test": "ember test"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test"
```

Comment:
Can you revert these 2 lines that `ember-cli-update` changed?

In this project, the `.github/workflows/ci.yml` and `testem.js` have been set up so that the `lint` and `test` scripts do separate things. 🙂